docs/appendix/changelog.md

```diff
@@ -10,6 +10,242 @@ aliases:
+## October 2023
+
+### Added: Test an ML model in browser
+
+Now, after you upload and train your machine learning model, you can test its results in the **Data** tab.
+With this feature you can now:
+
+- Select an image in the **Data** tab and access the **run model** section to choose and run a model
+- View the results directly above the image
+- Refine models by iteratively tagging more images for training based on observed performance
+
+To use this update, the classifier must have been trained or uploaded after September 19 2023.
+Additionally, the current version of this feature exclusively supports classification models.
+
+## Added: PLC support
+
+Support for the [Revolution Pi line of PLCs](https://revolutionpi.com/) from KUNBUS has been integrated into the Viam platform in the form of a module.
+This collaboration allows you to leverage the Raspberry Pi-based Revolution Pi, which runs on Linux and has a [specially designed I/O modules](https://www.raspberrypi.com/products/compute-module-4/?variant=raspberry-pi-cm4001000) for streamlined interaction with industrial controls, eliminating the need for additional components.
+
+Read the blog post for a step-by-step guide on using a PLC with Viam.
+
+### Improved: SLAM map creation
+
+The [Cartographer-module](/services/slam/cartographer/) now runs in Viam's cloud for creating or updating maps.
+This enhancement allows you to:
+
+- Generate larger maps without encountering session timeouts
+- Provide IMU input to improve map quality
+- Save maps to the **SLAM library** tab
+- Create or update maps using previously captured LiDAR and IMU data
+- Deploy maps to machines
+
+## September 2023
+
+### Added: Modular registry
+
+The Modular Registry enables you to use, create, and share custom modules, extending the capabilities of Viam beyond the components and services that are natively supported.
+
+This new feature allows you to:
+
+- Publish modules on the registry
+- Add modules to any machine's configuration with a few clicks
+- Select the desired module version for deployment, make changes at your convenience, and deploy the updates to a single machine or an entire fleet.
+
+### Added: Mobile app
+
+Now, you can use a mobile application, available for download now in the [Apple](https://apps.apple.com/us/app/viam-robotics/id6451424162) and [Google Play](https://play.google.com/store/apps/details?id=com.viam.viammobile&hl=en&gl=US) app stores, to connect to and control your Viam-powered machines directly from your mobile device.
+
+### Added: Power sensor component
+
+You now have the capability to use a power sensor component to monitor the voltage, current, and power consumption within your machine's system.
+
+To integrate your power sensor into Viam, you can [configure a module](/registry/configure/), or select from the following supported models:
+
+- [INA219](/components/power-sensor/ina219/)
+- [INA226](/components/power-sensor/ina226/)
+- [Renogy](/components/power-sensor/renogy/)
```

Comment:
this is good but could go out of date and cause people to think we don't support some models that we may have added by the time they read this. Better to just link to the docs:

```suggestion
You now have the capability to use a [power sensor component](/components/power-sensor/) to monitor the voltage, current, and power consumption within your machine's system.
```